### PR TITLE
Fixes #96 - Add team HTML and CSS

### LINKED ISSUE #96: HTML/CSS - Team - est. 1 day

### PR DESCRIPTION
**Pull request by Ctannerweb**

### CODE REVIEW

**Comment by Ctannerweb** [comment]
@acacheung :eyes: please :)

File: 2016/styles/modules/buttons.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.cta-btn {
+  padding: 19px 50px;
+  display: inline-block;
+  border: 2px solid var(--maine);
+  font-family: var(--grotesque);
+  font-size: 18px;
+  font-weight: var(--x-bold);
+  text-transform: uppercase;
+  color: var(--maine);
+  &:hover {
```
Comment (acacheung) [inline 2016/styles/modules/buttons.css]: hover and focus?

**Comment by acacheung** [comment]
hover state on link?

File: team.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        </symbol>
+        <symbol id="youtube" viewBox="0 0 1451 1024">
+          <path d="M575.569 700.847l-0.067-409.074 391.976 205.25-391.91 203.824zM1436.179 220.907c0 0-14.175-100.329-57.669-144.51-55.164-57.989-116.996-58.275-145.352-61.67-203-14.727-507.51-14.727-507.51-14.727h-0.631c0 0-304.503 0-507.51 14.727-28.357 3.395-90.167 3.682-145.351 61.67-43.495 44.181-57.649 144.51-57.649 144.51s-14.507 117.818-14.507 235.636v110.454c0 117.816 14.507 235.634 14.507 235.634s14.154 100.33 57.649 144.511c55.183 57.987 127.671 56.154 159.955 62.232 116.056 11.169 493.223 14.625 493.223 14.625s304.824-0.462 507.824-15.188c28.356-3.394 90.189-3.682 145.352-61.669 43.494-44.181 57.669-144.511 57.669-144.511s14.488-117.818 14.488-235.634v-110.454c0-117.819-14.488-235.636-14.488-235.636v0z"></path>
+        </symbol>
```
Comment (acacheung) [inline team.html]: soundcloud, spotify and youtube only needed on performance page. remove?

File: team.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        <a class="nav__link spot-color" href="attend.html">Attend</a>
+      </nav>
+    </header>
+    <main class="main team">
+      <svg class="heading__icon"><use xlink:href="#flag"></svg>
+      <h1 class="heading">Team</h1>
+      <div class="persons">
+        <figure class="person">
+          <img class="person__image" src="2016/images/team/maria-metveeva.jpg" alt="Maria Metveeva">
+          <figcaption class="person__name">Maria Metveeva</figcaption>
```
Comment (acacheung) [inline team.html]: Matveeva! hehe

File: team.html (hunk 2 above)
Comment (Ctannerweb) [inline team.html]: :+1: I also have to add a team page to each location and add in the location svg

**Pull request merged**